packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -396,21 +401,23 @@ Content-Type: text/json;charset=UTF-8
 }
 ```
 
-### Update a specific User (PATCH)
+### Update a specific User profile (PATCH)
 
-![Flowchart - update User (PATCH)](/img/oin/scim_flow-user-update-patch.png "Simple flow diagram for updating a User with a PATCH operation")
+![Flowchart - update User (PATCH)](/img/oin/scim_flow-user-update-patch.png "Simple flow diagram for updating a User with a PATCH method request")
 
 **PATCH** /Users/*$userID*
 
-A User object can be updated through a PATCH operation for the following actions:
+For new OIN app integrations, the following operations update a User object through a PATCH method request:
```

Comment:
"User object" or "User profile object"?

Reply:
Added text to clarify "user application profile" (Okta side) vs "User object" (SCIM server side) throughout.